firmware/src/main.cpp

```diff
@@ -84,6 +87,7 @@ void setup()
 
 void loop()
 {
+    // PROFILE("Test Stand", 1000, test_stand.execute());
     test_stand.execute();
 
     // Blink an LED
```

Comment:
Is this functionality to visually get an idea if serial communication is working? Just a note that with the shield installed, we won't be able to see the LED blink :(

Reply:
This doesn't tell us anything about serial communication necessarily, just lets me know the program hasn't stalled or crashed. You can sorta kinda see it with the shield if you look at it sideways right?